src/components/AnchorForCommentsOnly/anchorForCommentsOnlyPropTypes.js

```diff
@@ -24,6 +24,8 @@ const propTypes = {
     /** Any additional styles to apply */
     // eslint-disable-next-line react/forbid-prop-types
     style: PropTypes.any,
+
+    text: PropTypes.string,
```

Comment:
Move this prop above style, and add comment
Also rename from text to fileName

Reply:
Done